src/wp-admin/includes/class-wp-automatic-updater.php

```diff
@@ -131,7 +131,29 @@ public function is_vcs_checkout( $context ) {
 	 *
 	 * @param string $type    The type of update being checked: 'core', 'theme',
 	 *                        'plugin', 'translation'.
-	 * @param object $item    The update offer.
+	 * @param object $item {
+	 *     The update offer.
+	 *
+	 *     @type string   $current            The version number of the update offer.
```

Comment:
This appears to represent the current (latest) version within the major version range being offered. So for example, right now 5.8.2 is the latest version for 5.8. If an update was being returned for 5.8.1, this would be set to 5.8.2.